ui-testing-core/src/main/java/com/avito/android/test/action/LibraryViewActions.kt

```diff
@@ -0,0 +1,22 @@
+package com.avito.android.test.action
+
+import android.support.test.espresso.ViewAction
+import android.support.test.espresso.action.ViewActions
+import com.avito.android.test.UITestConfig
+
+internal object LibraryViewActions {
+
+    fun click(): ViewAction = when (UITestConfig.clickRollbackPolicy) {
```

Comment:
Can we pass a rollback policy as a parameter of click? 
In that case we could override the default policy if we need it only in some interactions.